### ISSUE TITLE
Make the default channel configurable via an environment variable

### ISSUE THREAD

**Issue by lassepe** [JuliaLang/juliaup#375]
I would like to migrate from `asdf-vm` over to juliaup. The only missing feature for me to do that would be to be able to set a channel per directory. I don't think that the feature of setting a version per directory should be handled by juliaup itself because it seems out of scope. Instead, I think it would be nice to use something like `direnv` to export an environment variable like `JULIAUP_CHANNEL`. Upon start of the julia launcher, the launcher can check if this environment variable is set and use it as a default channel if the user does not request a specific channel via `julia +channel`.

**Comment by davidanthoff** [comment]
Yes, entirely agreed. I think we _also_ want something file and Julia specific, see https://github.com/JuliaLang/juliaup/issues/10. At the end we'll need something like the rustup hierarchy of configuration locations for this. But one should definitely be an env variable, and then that should make it all work easily with `direnv` for those that want to go that route.

**Comment by asinghvi17** [comment]
Looks like this got added in https://github.com/JuliaLang/juliaup/commit/5414b75 and has been in quite a few releases.  It's available via the JULIAUP_CHANNEL env var.

Can be closed I think.
